src/patternfly/components/Page/page.scss

```diff
@@ -132,9 +132,9 @@
   --pf-c-page__main-breadcrumb--PaddingRight: var(--pf-global--spacer--md);
   --pf-c-page__main-breadcrumb--PaddingBottom: 0;
   --pf-c-page__main-breadcrumb--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-page__main-breadcrumb--md--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-page__main-breadcrumb--md--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-page__main-breadcrumb--md--PaddingLeft: var(--pf-global--spacer--xl);
+  --pf-c-page__main-breadcrumb--md--PaddingTop: var(--pf-global--spacer--lg);
```

Comment:
Sidebar nav and the breadcrumb/page-nav aren't lining up now.

<img width="1170" alt="Screen Shot 2019-04-25 at 5 12 26 PM" src="https://user-images.githubusercontent.com/20118816/56768819-5ff55f80-677d-11e9-9a13-691ee90c1692.png">

Reply:
Yeah that's a tricky one. The main content sections don't line up, either. Tertiary nav does though.

We could make whatever section is first (breadcrumbs, tertiary nav, main content section) have padding that aligns it with the sidebar, or we could move the nav up a little, then update the tertiary nav to match. 

What do you think?

Reply:
@mcoker We should move the sidebar nav up a little to match.